envision/web/src/index.js

```diff
@@ -26,7 +26,7 @@ import App from "./components/app.js";
 import "antd/dist/antd.dark.css";
 
 let client = new Client({
-  endpoint: window.location.origin,
+  endpoint: "http://localhost:8081",
```

Comment:
why are you switching this back? did the `location.origin` not work for you?

Reply:
It didn't. `npm run` puts it on `8080` while the server always remains on `8081`.